examples/particles/fireworks/fireworks.lua

```diff
@@ -0,0 +1,56 @@
+local M = {}
+
+local _tension = 0.9
+local _gravity = 980
+
+function M.start_fireworks(trail_id, bang_id, start_pos, speed, time, delay, gravity, tension)
```

Comment:
I don't think it helps to understand the example when the start_fireworks() function is in a Lua module like this. I would actually prefer if this could be a local function in fireworks.script instead.

Reply:
ok, got it